src/components/Voting/VotingCard.tsx

```diff
@@ -4,15 +4,31 @@ import RemoveMarkdown from 'remove-markdown'
 import Card from 'components/Card'
 import FormattedDate from 'components/Voting/FormattedDate'
 
-import { VotingProposalProps } from 'models/voting'
+import {
+  VotingProposalProps,
+  VotingResultStatus,
+  VotingStatus,
+} from 'models/voting'
 
 interface VotingCardProps {
   proposal: VotingProposalProps
 }
 
 const VotingCard: React.FC<VotingCardProps> = ({
-  proposal: { state, title, body, start, id },
+  proposal: { state, title, body, start, id, choices, scores },
 }) => {
+  let stateText = state
+  if (state === VotingStatus.CLOSED) {
+    const highestVotedIndex = scores?.indexOf(Math.max(...scores))
+    if (choices[highestVotedIndex] === 'For') {
```

Comment:
could you also make an enum for these choices? `For`, `Against`, and `Abstain`

```
export enum VotingResultStatus {
  FOR = 'For',
  AGAINST = 'Against',
  ABSTAINED = 'Abstain',
}

Reply:
hey @niikkiin - sure thing! will make the changes.

Reply:
Thank you @mitesh-mutha!!